packages/artillery/lib/util.js

```diff
@@ -98,11 +98,12 @@ function addDefaultPlugins(script) {
   return finalScript;
 }
 
-async function resolveConfigTemplates(script, flags, configPath) {
+async function resolveConfigTemplates(script, flags, configPath, scriptPath) {
```

Comment:
We add the additional argument here for the path to script (file that holds `scenarios`) and set it to `vars` making it available at config level